### ISSUE TITLE
An ADA issue: the clearance prices (red) have an insufficient color contrast

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#7]
![04](https://user-images.githubusercontent.com/83204/46413093-07f3fe80-c742-11e8-9591-04249db786bf.png)

![01](https://user-images.githubusercontent.com/83204/46413104-0c201c00-c742-11e8-9141-1915e073cc9e.png)

**Comment by dmitry-fedyuk** [comment]
Fixed in [0.0.5](https://github.com/kitchencollection-com/core/releases/tag/0.0.5).
